app/main.py

```diff
@@ -2,4 +2,57 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        self.name = name
```

Comment:
you can specify here:

```suggestion
    def __init__(self, name: str, weight: int, coords: list[int] = [0, 0]) -> None:
        self.name = name
```

Reply:
if I fix the code as you indicated, the tests will fail